### PR TITLE
Create CONTRIBUTING.md

### LINKED ISSUE #69: Create `CONTRIBUTING.MD`
Create `CONTRIBUTING.MD`, for example like the [Atom’s](https://github.com/atom/atom/blob/master/CONTRIBUTING.md).

### PR DESCRIPTION
**Pull request by ahmetcetin**
closes #69

### CODE REVIEW

**Comment by tukusejssirs** [comment]
Thanks, @ahmetcetin! However, the current dev branch is `romcal-es6`—could you merge it into that branch? :)

Anyway, do you how to create an `npm` build? We currently need a developer who would solve #49 as currently, the dev branch is not working and the repo owner (@pejulian) is busy in long-term. There also some deprecated packages that should be upgraded/replaced. Thanks in advance. :)

**Comment by emagnier** [comment]
Also when it get ready, we will merge it into `master`.
The `romcal-es6` branch is currently not stable and have blocking issue.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+Please do not be afraid of contributing code, make sure it follows these rules:
+
+* Your code compiles, does not break any of the existing code in the master branch and does not cause conflicts,
+* The code is readable and has comments, that aren’t superfluous or unnecessary,
+* An overview or context is provided as body of the Pull Request. It does not need to be too extensive.
+
+Extra points if your code comes with tests!
+
+For more information please check [README](README.md)
```
Comment (ahmetcetin) [inline CONTRIBUTING.md]: added link to readme

Comment (emagnier) [reply CONTRIBUTING.md]: I mean, add instead a link **in** the `README.md` file, -> **to** `CONTRIBUTING.md`.
`README.md` is displayed in the github repository, it's like the starting point for all documentation.

**Comment by tukusejssirs** [comment]
I don’t know if `CONTRIBUTING.md` is the right place, but we should list the names of branches and tell the users / potential contributors which branch is branch, like:
- `master` is the main/default branch; use this branch for PRs;
- `romcal-es6` is old dev branch;
- `jarosz` is old branch that still might contain new features not implemented yet into `master`.

**Comment by emagnier** [comment]
`romcal-es6` should be deleted when things will be ready and merged into master. And `jarosz` is a 5 years old branch and we should just ignore it.

By default Github will create new PR by targeting the `master` branch (and this is a common practice for  many projects hosted on Gihtub). In the future if a PR introduce major code/feature change, we will see case by case. But for the time beeing I think it's not worth mentioning which branch we should use (to avoid a risk of information overload).

File: CONTRIBUTING.md (hunk 1 above)
Comment (tukusejssirs) [reply CONTRIBUTING.md]: @ahmetcetin, could you, please, remove `For more information please check [README](README.md)` from `CONTRIBUTING.md` and create new section in the `README.md` file named `Contributing`, where you would create link to `CONTRIBUTING.md`?

Thanks! :smiley:

Comment (ahmetcetin) [reply CONTRIBUTING.md]: updated readme and contributing as per your request.
Cheers,
Ahmet

**Comment by emagnier** [comment]
Thanks. I've cherry-picked your commit from your other pull request #89, so they are merged together.

**Pull request merged**